actions/class.Tests.php

```diff
@@ -155,6 +155,7 @@ public function delete()
         }
 
         $uri = $this->getRequestParameter('id');
+        $classUri = $this->getRequestParameter('classUri');
```

Comment:
What about other types of resources? Like Item class deletion that is reported in the Task?

Reply:
I checked this for Item class deletion and everything works fine 🙂 
There's an update in task: `User is able to find Tests (and it is only reproduced for tests)`.